requirements-test.txt

```diff
@@ -0,0 +1,6 @@
+# Constraints on requirements below
+-c constraints.txt
```

Comment:
Does this work due to the way `-r` handles `requirements-test.txt`?
Could we have `pip install -r requirements-test.txt -c constraints.txt` in the install section of .travis.yml instead?

Reply:
See https://github.com/apel/apel/pull/171#discussion_r220097100.

Reply:
fair enough